lcm_types/mode_input.lcm

```diff
@@ -0,0 +1,4 @@
+struct ModeInput
+{
+  int16_t    mode;
```

Comment:
Should probably be a different PR?

Reply:
Agreed, should probably rebase a81f6688a616721fc89d664e42eecd35c83525ee off of fd4b851c454f5986c7b01015873a63c63686b3dd

Reply:
See #24